From ef04630df0791dc9247e8f24d60af39dc01ec416 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 21 Dec 2023 16:03:01 +0100
Subject: [PATCH] com.webos.service.bluetooth2.role.json.in: Add additional
 permissions

Fix LS2 permission issues:

Dec 21 14:56:10 qemux86-64 ls-hubd[242]: [] [pmlog] ls-hubd LSHUB_NO_OUT_PERMS {"DEST_APP_ID":"com.webos.service.mediacontroller","SRC_APP_ID":"com.webos.service.bluetooth2","EXE":"/usr/sbin/webos-bluetooth-service","PID":1023} "com.webos.service.bluetooth2" does not have sufficient outbound permissions to communicate with "com.webos.service.mediacontroller" (cmdline: /usr/sbin/webos-bluetooth-service)

Dec 22 08:49:25 pinephonepro ls-hubd[341]: [] [pmlog] ls-hubd LSHUB_NO_OUT_PERMS {"DEST_APP_ID":"com.webos.monitor1594","SRC_APP_ID":"com.webos.service.bluetooth2","EXE":"/usr/sbin/webos-bluetooth-service","PID":689} "com.webos.service.bluetooth2" does not have sufficient outbound permissions to communicate with "com.webos.monitor1594" (cmdline:
/usr/sbin/webos-bluetooth-service)

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending 

 files/sysbus/com.webos.service.bluetooth2.role.json.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.service.bluetooth2.role.json.in b/files/sysbus/com.webos.service.bluetooth2.role.json.in
index b2d5e74..f7145d5 100644
--- a/files/sysbus/com.webos.service.bluetooth2.role.json.in
+++ b/files/sysbus/com.webos.service.bluetooth2.role.json.in
@@ -6,7 +6,7 @@
     "permissions": [
         {
             "service":"com.webos.service.bluetooth2",
-            "outbound":["com.webos.service.account" , "com.webos.service.pdm", "com.webos.service.db"]
+            "outbound":["com.webos.service.account" , "com.webos.service.pdm", "com.webos.service.db", "com.webos.service.mediacontroller", "com.webos.monitor*"]
         }
     ]
 }
